doc-content/enterprise-only/installation/clustering-dev-con.adoc

```diff
@@ -1,16 +1,11 @@
 [id='clustering-dev-con_{context}']
 = {PRODUCT} clusters in a development (authoring) environment
 
-[NOTE]
-====
-Configuration of {CENTRAL} for high availability is currently technology preview.
-====
-
 ifdef::PAM[]
 Developers can use {PRODUCT} to author rules and processes that assist users with decision making.
 
 endif::[]
-You can configure {PRODUCT} as a clustered development environment to benefit from high availability. With a clustered environment, if a developer is working on `$node1` and that node fails, that developer's work is preserved and visible on any other node of the cluster.
+You can configure {PRODUCT} as a clustered development environment to benefit from high availability. With a clustered environment, if a developer is working on the first node and that node fails, that developer's work is preserved and visible on any other node of the cluster.
```

Comment:
Again, up to you. I suggest minor changes for:

"if a developer is working on the first node and that node fails, that developer's work is preserved and visible on any other node of the cluster."

if a developer is working a node that fails, the developer's work is preserved and visible on any other node in the cluster.

Reply:
Done